libiocage/lib/Jail.py

```diff
@@ -601,7 +797,14 @@ def __getattr__(self, key):
 
         raise AttributeError(f"Jail property {key} not found")
 
-    def getattr_str(self, key):
+    def getstring(self, key):
```

Comment:
why is this a public function rather than a `__str__` on the properties?
or would that be too convoluted?